control-plane/agents/core/src/volume/tests.rs

```diff
@@ -19,7 +19,7 @@ use common_lib::{
     },
 };
 
-use rpc::mayastor::FaultNexusChildRequest;
+use composer::rpc::mayastor::FaultNexusChildRequest;
```

Comment:
We should remove the rpc from the composer library, the rpc should be separate now otherwise if we update the rpc we also may have to update the composer?

Reply:
Composer currently exposes RPC access to containers (via gRPC handles or so). If we keep this functionality, then it's more useful to have rpc in this crate too, as for user it's more convenient to just use composer crate to access RPC. If we decide to completely remove RPC support from composer crate (in contrary to what we have now), than we can remove rpc, sure.

Reply:
Yeah it's indeed convenient but also because it was in the same repo as the rpc which is no longer the case now as it's a "common" interface to docker.
We can have our own wrapper for it, as long as the composer exposes the endpoint information.